google/cloud/spanner_v1/pool.py

```diff
@@ -235,6 +369,26 @@ def get(self, timeout=None):
             session = self._database.session()
             session.create()
 
+        # Set session properties.
+        session.checkout_time = datetime.datetime.utcnow()
+        session.long_running = isLongRunning
+        session.transaction_logged = False
+        self._borrowed_sessions.append(session)
+        self._traces[session._session_id] = traceback.extract_stack()
+
+        # Start background task for handling long-running transactions if 95% of sessions are checked out.
+        if (
+            self._database.close_inactive_transactions or self._database.logging_enabled
+        ) and len(self._borrowed_sessions) / self.size >= 0.95:
+            self.startCleaningLongRunningSessions()
+
+        # Log a warning message if Session pool is exhausted.
+        if (
```

Comment:
Should be if else for 100% and 95%, otherwise two logs will be printed

Reply:
Done